client/components/explorer/explorer-tab-service.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @copyright Copyright 2014 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @fileoverview Service for state and content of Explorer tabs.
+ * @author joemu@google.com (Joe Allan Muharsky)
+ */
+
+goog.provide('p3rf.perfkit.explorer.components.explorer.ExplorerTabService');
+goog.provide('p3rf.perfkit.explorer.components.explorer.ExplorerTabModel');
+
+
+
+goog.scope(function() {
+var explorer = p3rf.perfkit.explorer;
```

Comment:
This file is actually an unused duplicate of explorer/sidebar/sidebar-tab-service.js.  I thas been removed.